packages/plugin-value-grid/package.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "@crypric-css/plugin-value-ratio",
```

Comment:
Grid will be a value plugin.
Value plugins will are manipulating value only. Prop-plugins are adding custom props. Should we call plugin the previous and extension the last instead?